beacon_node/src/cli.rs

```diff
@@ -233,6 +233,27 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .takes_value(true),
         )
 
+        /*
+         * Explorer metrics
+         */
+
+         .arg(
+            Arg::with_name("explorer-metrics")
```

Comment:
Are the `explorer-metrics` and `explorer-address` flags standard between clients? Just out of interest.

Reply:
Nope that's not standard. Can be anything.